packages/blockchain-wallet-v4-frontend/src/data/components/sendCrypto/sagas.ts

```diff
@@ -124,6 +149,120 @@ export default ({ api }: { api: APIType }) => {
     }
   }
 
+  // helper function to send BTC amount
+  const submitBTCTransaction = function* () {
```

Comment:
We shouldn't need a submitCOINTransaction function, it should be coin agnostic

Reply:
yeah this is for testing purpose to try to make at least BTC working then will move it to generic function